DataFormats/EcalDigi/interface/EcalTimeDigi.h

```diff
@@ -0,0 +1,59 @@
+#ifndef ECALTIMEDIGI_H
+#define ECALTIMEDIGI_H 1
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h